setup.cfg

```diff
@@ -0,0 +1,3 @@
+[flake8]
+exclude = __init__.py, ./doc/*
+ignore = E501, F405
```

Comment:
Since we have setup.cfg now, maybe we could move pytest.ini in here?
It needs config under: `[tool:pytest]` header